### PR TITLE
Add additional values to enhance flexibility of helm-exporter

### PR DESCRIPTION
**Pull request by paulhopkins11**
#### Is this a new chart
No, this is an enhancement to @sstarcher chart
> NOTE: We're experiencing a high volume of PRs to this repo and reviews will be delayed. Please host your own chart repository and submit your repository to the Helm Hub instead of this repo to make them discoverable to the community. [Here](https://github.com/helm/hub/blob/master/Repositories.md) is how to submit new chart repositories to the Helm Hub.

#### What this PR does / why we need it:

When deploying this chart into our production cluster we found it did not contain the ability to specify imagePullSecrets as well as annotations to allow kube2iam to be used. 

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by sstarcher** [comment]
Why would you want to use kube2iam with helm-exporter?  It does not make any AWS IAM calls.

File: stable/helm-exporter/templates/deployment.yaml [hunk 1]
```diff
@@ -2,11 +2,16 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: {{ include "helm-exporter.fullname" . }}
+  namespace: {{ .Release.Namespace }}
```
Comment (sstarcher) [inline stable/helm-exporter/templates/deployment.yaml]: Helm supplies this does it not?  Why do we need to specify this value?

Comment (paulhopkins11) [reply stable/helm-exporter/templates/deployment.yaml]: Helm will supply on an install. This allows helm package to be used. Followed the convention from https://github.com/helm/charts/blob/master/stable/grafana/templates/deployment.yaml
Happy to remove, this isn't critical for us

Comment (sstarcher) [reply stable/helm-exporter/templates/deployment.yaml]: Not sure what you mean by `allows helm package to be used`

**Pull request merged**